src/Xamarin.Android.Tools.AndroidSdk/JdkInfo.cs

```diff
@@ -258,6 +259,12 @@ static bool AnySystemJavasInstalled ()
 					const string NameValueDelim         = " = ";
 					if (string.IsNullOrEmpty (e.Data))
 						return;
+					if (e.Data.StartsWith ("Property settings:", StringComparison.Ordinal)) {
```

Comment:
So probably what happened, is some JDK doesn't understand `-XshowSettings:properties` and ignored it?

This seems like the right string to look for:

https://github.com/openjdk/jdk/blob/739769c8fc4b496f08a92225a12d07414537b6c0/test/jdk/tools/launcher/Settings.java#L68